imdown-mobile/squads/add_squad.tsx

```diff
@@ -61,17 +64,32 @@ const AddSquadModal = (props: OwnProps) => {
           visible={props.visible}
           presentationStyle={"formSheet"}
         >
-          <View style={AddSquadStyles.nameEmojiContainer}>
-            {renderEmoji()}
-            <TextInput placeholder={"add squad name"} onChangeText={(name) => setSquadName(name)} 
-              style={AddSquadStyles.squad_name}
-              placeholderTextColor = "lightgray"/>
+          <View style={AddSquadStyles.container}>
+            <SafeAreaView style={AddSquadStyles.emoji_and_squad_name_container}>
+              {renderEmoji()}
+              <TextInput placeholder={"add squad name"} onChangeText={(name) => setSquadName(name)} 
+                style={AddSquadStyles.squad_name}
+                placeholderTextColor = "lightgray"/>
+            </SafeAreaView>
+            
             <TouchableOpacity onPress={addSquad} style={AddSquadStyles.add_squad_button}>
               <Text style={AddSquadStyles.add_squad_text}>Done</Text>
             </TouchableOpacity>
 
+            {/* <View style={AddSquadStyles.addSquadButton}>
+              <TouchableOpacity onPress={addSquad} style={AddSquadStyles.add_squad_button}>
+                <Text style={AddSquadStyles.add_squad_text}>Done</Text>
+              </TouchableOpacity>
+            </View> */}
+
           </View>
 
+          {/* <View style={AddSquadStyles.addSquadButton}>
```

Comment:
no comments pls unless its explaining what the code is doing, just delet the code